src/PureCloud.Client/Apis/RoutingApi.cs

```diff
@@ -31,6 +32,8 @@ public async Task DeleteRoutingAssessmentAsync(string assessmentId, Cancellation
         var response = await client.DeleteAsync($"api/v2/routing/assessments/{assessmentId}", cancellationToken);
 
         response.EnsureSuccessStatusCode();
```

Comment:
If returning IsSucceasStatusCode then don't invoke EnaureSuccessStatusCode(). Do the same thing in all the methods